src/Tickets/Seating/app/frontend/ticketsBlock/index.js

```diff
@@ -230,21 +231,22 @@ function updateTotals(parentElement) {
  */
 function addTicketToSelection(parentElement, props) {
 	parentElement = parentElement || document;
-	const ticketPrice = tickets?.[props.ticketId]?.price || null;
+	const priceValue = tickets?.[props.ticketId]?.priceValue || null;
+	const formattedPrice = tickets?.[props.ticketId]?.price || null;
 	const ticketName = tickets?.[props.ticketId].name || null;
```

Comment:
Any reason this one doesn't have a `?` after ticketId?

Reply:
I assume it's been working fine all along, so approved - just asking

Reply:
It checks for null before moving onto the child property. If it doesn't exist, it fails with no error.